packages/column-views/src/maps/column-navigation/mapbox/state.ts

```diff
@@ -107,3 +110,16 @@ export function useColumnNavigationStore(
   }
   return useStore(storeApi, selector);
 }
+
+function getColsData({columns}) {
+  let res = useAPIResult(
+    "https://macrostrat.org/api/v2/columns?col_id=" + columns?.join(",") + "&response=long&format=geojson"
```

Comment:
This function should be deleted but also note that it's not usually ideal to hard-code a Macrostrat URL in this kind of location, as we might want to run this in different environments